src/website/views/custom-elements/expansion-panel.pug

```diff
@@ -0,0 +1,278 @@
+---
+title: Expansion panel
+scripts:
+  - js/ce/ux-chi-ce
+styles:
+  - assets/styles/expansion-panel-ce
+---
+
+p.-text
+  | Expansion panels are components designed to be folded and stacked in sequence. They have four states: active,
+  | pending, done and disabled. When pending, only the title is shown. When active, panels expand to show
+  | content or forms. When done, content or form data is presented in a read only state. When disabled, a muted title
+  | is shown in a non-clickable state. Three classes represent these states:
+  | <code>-done</code>, <code>-active</code>, and <code>-disabled</code>. Pending is the default state.
```

Comment:
Fix this text as we don't use classes in the custom elements version of Chi.

Reply:
Please, review this documentation and adapt it to Custom Elements version. Numerous references to Chi CSS has been found.